PEP 11: Clarify Fixed vs. Modern lifecycle & non-support of ESU (#2802)
python/peps: pep-0011.txt

Microsoft Windows
'''''''''''''''''

> Windows versions prior to Windows 10 follow Microsoft's `Fixed Lifecycle Policy <https://learn.microsoft.com/en-us/lifecycle/policies/fixed>`__,

Author: Oops, didn't notice there wasn't a hard break there, since the soft break in my Editor lined up at the same spot. Thanks for the catch!

with a mainstream support phase for 5 years after release,
where the product is generally commercially available,
and an additional 5 year extended support phase,
where paid support is still available and certain bug fixes are released.

> Extended support is distinct from `ESU (Extended Security Updates)
> <https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates>`_,
> a specialized, paid program available to high-volume enterprise customers
> as a "last resort" option after extended support ends.

Author: ```suggestion
`Extended Security Updates (ESU)
<https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates>`_
is a paid program available to high-volume enterprise customers
as a "last resort" option to receive certain security updates after extended support ends.
ESU is considered a distinct phase that follows the expiration of extended support.
```

Slight tweak to reflect that it isn't just our opinion, and rather MSFT itself certainly treats ESU as distinct from extended support and after nominal EoL.

> as a "last resort" option after extended support ends.

Author: Well, I suppose technically yeah, but in that case you could say the same thing about anything else in this paragraph, heh. The main intention of this one is to summarize the situation on MS's side, while the following paragraph ("CPython's Windows support...") discusses the opinion aspects and how CPython handles it.

Windows 10 and up follow Microsoft's `Modern Lifecycle Policy
<https://learn.microsoft.com/en-us/lifecycle/policies/modern>`__,
which varies per-product, per-version, per-edition and per-channel.
Generally, feature updates (1709, 22H2) occur every 6-12 months
and are supported for 18-36 months;
Server and IoT editions, and LTSC channel releases are supported for 5-10 years,
and the latest feature release of a major version (Windows 10, Windows 11)
generally receives new updates for at least 10 years following release.
Microsoft's `Windows Lifecycle FAQ
<https://learn.microsoft.com/en-us/lifecycle/faq/windows>`_
has more specific and up-to-date guidance.

CPython's Windows support currently follows Microsoft's lifecycles.
A new feature release X.Y.0 will support all Windows versions

> whose extended support phase is not yet expired
> (ESU is not considered supported).

Author: Actually, that only works on Markdown, `*` works on both md and reST. So:

```suggestion
whose *extended support* phase has not yet expired.
```

Subsequent bug fix releases will support the same Windows versions

> as the original feature release, even if no longer supported by Microsoft.

Author: ```suggestion
as the original feature release, even if no longer supported by Microsoft.
New versions of Windows released while CPython is in maintenance mode
may be supported at the discretion of the core team and release manager.
```

Tweaked a word for clarity/grammar
